README.md

```diff
@@ -94,13 +94,13 @@ We vendor some data from other repositories that you will sometimes need to upda
 
 ### Deploying Emission
 
-**Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break.
+**Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break. To release an update to bump Eigen's Emission version, you must use either a major or minor release.
```

Comment:
I’m not sure I understand what this is about, it’s fine to do a patch release for inclusion in Eigen. What gives?